.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
 cache: pip
 python:
-  - "nightly" # currently points to 3.6-dev
+  # see https://docs.travis-ci.com/user/languages/python/ for options
+  - "nightly"
```

Comment:
How's 3.7 the same as 3.6-dev? (I don't know much about Python's dev cycle and after reading https://docs.python.org/devguide/devcycle.html, it seems to be saying that the default branch == 3.7 != 3.6)

Is 3.6-dev https://hg.python.org/cpython/shortlog/3.6?